README.md

```diff
@@ -231,8 +231,26 @@ To edit your public profile:
     * The background and highlight colors in which your public profile should be displayed.
 
 ### Your storage
-Even if you have navigated to a different Pod, you can still view your storage.
+
+The **Your storage** menu item brings you to an overview of all the resources within your Pod. This will always display the storage of your WebID, so it is your personal storage. Even if you navigate to a different Pod, you can still only view your personal storage.
 
 To view your storage:
 1. Select **Your storage** from the dropdown menu.
-2. A Folder view of your storage is displayed.
+2. A folder view of your storage is displayed.
+
+To visualize and have access to the storage of another user, you have two options, at the moment:
+
+1. Add the storage directly in your profile/card.
+
+   Your storage is connected to your profile with a triple like `:me space:storage </> . `. You can simply add a second triple with the same predicate, like `:me space:storage <otherWebID>`, to your WebID Profile Document (that is, in your file that can be found under a URI like this one: `<https://timea.solidcommunity.net/profile/card#me>`). An example of the triple to be added is `:me space:storage <https://suser.solidcommunity.net> .`
+
+_**remark:** Make sure you have the correct prefix for `space:storage`. The full URI is `<http://www.w3.org/ns/pim/space#storage>`.
+_**remark:** You will only be able to visualize the resources that are publicly readable in the second Pod, unless you have full access to the second Pod.
```

Comment:
```suggestion
_**remark:** You will only be able to visualize the resources that are readable with your WebID in the available Pod's.
```

Reply:
+1 to @bourgeoa's suggestion, modulo plural `Pod's` should have no apostrophe, so should be `Pods`.